apps/web/src/components/createFormTemplate/CreateFormTemplateModal.tsx

```diff
@@ -55,6 +56,7 @@ export const CreateFormTemplateModal = ({
 
   const [pdf, setPdf] = useState<string | ArrayBuffer | null>(null);
   const [pdfName, setPdfName] = useState<string | null>(null);
+  const [pdfFile, setPdfFile] = useState<File | null>(null);
```

Comment:
What I'm noticing is that this state is not getting reset after a file upload, for instance if you create a template where you load in a template and try to upload another template the previous state is still there, which allows you to bypass the !pdfFile check as well although the form preview is blank, could we reset this state?